src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## Unreleased
 
+* Introduced a new metric, `http.server.request.duration`, for users who opt-in
```

Comment:
Introduced a new metric, `http.server.request.duration`, for users who opt-in
to the new semantic convention by configuring the `OTEL_SEMCONV_STABILITY_OPT_IN`
environment variable to value "fillthis". This metric measures time in **seconds**, but histogram buckets are adjusted
automatically for this metric by OTel SDK which [specially treat this](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4820) metric. This metric follows the OTel Semantic Conventions from (add link).
  * Metric Name: `http.server.request.duration`
  * Unit: `seconds`
  * Histogram Buckets: `0, 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5,
    0.75, 1,  2.5, 5, 7.5, 10` (due to OpenTelemetry SDK's special treatment)
  * Replaces old metric: `http.server.duration`
    * Unit: `miliseconds`
    * Histogram Buckets: `0, 5, 10, 25, 50, 75, 100, 250, 500, 750, 1000, 2500,
    5000, 7500, 10000` (the general default of Histogram buckets)

`OTEL_SEMCONV_STABILITY_OPT_IN` may be set to "fillthis", to help an easier transition, at the cost of emitting both old and new metrics.

^did some refactoring. Feel free to take it with modifications. 
A user who do not have any experience with OTel sem. conventions and the new vs old, should find it easy to follow the changelog. Key things I'd prefer to convey here are:
1. No breaking changes if you just update to this new version.
2. Opt-in feature to emit a new metric that follows a new convention, which despite using "s", still allows good histogram precision due to SDK treating it specially.
3. Hint at emitting both to smooth transition.
4. Hint that the new metric is the long term bet, as it'll be the one getting stabilizied by Otel sem convention.

Reply:
Thanks for the refactor. I'll take another pass at this today.

Do we typically put instructions in the changelog? I was trying to keep the changelog succinct, and use the readme for the full information. The challenge is that this change has a LOT of details :)

Reply:
just pushed my rewrite. I think I've addressed all your points. please re-review.